### ISSUE TITLE
v0.4.0 EP104 CUDA - low hash on 3070

### ISSUE THREAD

**Issue by megalolka** [Qubic-Solutions/rqiner-builds#31]
Hi!
Windows version of CUDA 0.4.0 gives 65 its on 3060 12Gb and only 50 its on 3060ti and 3070 (8 Gb both).
And power load of 3060ti \ 3070 is less that on 3060. 
Please fix this issue.

**Comment by Arschmole** [comment]
This is expected. Depending on the hardware, the performance (it/s) in EPOCH 104 is around 1/4 - 1/3 of the performance of EPOCH 103.

**Comment by megalolka** [comment]
No, this is a bug, when more powerful 3070 has lower hashrate than 3060:
3060 = 64 its
3060ti and 3070 = 50 its
I switched to other miner (Eloword) and is works correctly:
3060 = 43 its
3070 = 69 its

Waiting for fix, then I come back to rqiner miner.

**Comment by Arschmole** [comment]
3060 has 12GB of RAM, 3060 Ti and 3070 only 8GB.

**Comment by megalolka** [comment]
> 3060 has 12GB of RAM, 3060 Ti and 3070 only 8GB.

Say it to Eloword. His miner works correctly!
